### ISSUE TITLE
[BUG] crc setup fails with  yum: command not found

### ISSUE THREAD

**Issue by T0MASD** [crc-org/crc#3452]
### General information

  * OS: Linux / macOS / Windows
  * Hypervisor: KVM / Hyper-V / hyperkit / vfkit
  * Did you run `crc setup` before starting it (Yes/No)?
  * Running CRC on: Laptop / Baremetal-Server / VM

## CRC version
```bash
$ crc version
CRC version: 2.11.0+823e40d
OpenShift version: 4.11.13
Podman version: 4.2.0
```
  

## Host Operating System
```bash
NAME="Fedora Linux"
VERSION="38 (Rawhide Prerelease)"
ID=fedora
VERSION_ID=38
VERSION_CODENAME=""
PLATFORM_ID="platform:f38"
PRETTY_NAME="Fedora Linux 38 (Rawhide Prerelease)"
ANSI_COLOR="0;38;2;60;110;180"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:38"
DEFAULT_HOSTNAME="fedora"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/rawhide/system-administrators-guide/"
SUPPORT_URL="https://ask.fedoraproject.org/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=rawhide
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=rawhide
```

### Steps to reproduce

  1. crc setup
  
### Expected

crc detects dnf package manager on fedora and ues it

### Actual
```
INFO Using root access: Installing virtualization packages 
Could not install required packages:  exit status 127: /bin/sh: line 1: yum: command not found
```

**Comment by praveenkumar** [comment]
@T0MASD does fedora going to drop yum => dnf script which as of now call dnf if user execute `yum` command?

**Comment by anjannath** [comment]
Fixed via #3454
